tests/cross_functional/system_test/test_full_cluster_health.py

```
@@ -134,17 +129,46 @@ def delete_pods(self):
+        # Wait for Ceph to recover from HEALTH_ERR
+        logger.info(
+            f"Waiting up to {timeout} seconds for Ceph to recover from HEALTH_ERR state"
+        )
+        sample = TimeoutSampler(
+            timeout=timeout, sleep=30, func=self.ceph_cluster.get_ceph_health
+        )
+
+        try:
+            for ceph_status in sample:
+                if "HEALTH_ERR" not in ceph_status:
+                    logger.info(f"Ceph recovered successfully. Status: {ceph_status}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)